tools/update_bt_diagrams.bash

```diff
@@ -9,3 +9,6 @@ navigation2/tools/bt2img.py \
 navigation2/tools/bt2img.py \
   --behavior_tree navigation2/nav2_bt_navigator/behavior_trees/navigate_w_replanning_and_recovery.xml \
   --image_out navigation2/nav2_bt_navigator/doc/parallel_w_recovery
+navigation2/tools/bt2img.py \
```

Comment:
Please also check-in the diagram, and add a quick section to the bt_navigator readme that describes the purpose and references the diagram, like we have for the other trees.